packages/elements/src/utils/createNewElement.js

```diff
@@ -37,9 +36,11 @@ const createNewElement = (type, attributes = {}) => {
     type,
     id: uuidv4(),
   };
+
   // There's an exception for the background shape that should not get all the default attributes.
   if (attributes.isDefaultBackground) {
-    return objectWithout(newElement, ['backgroundColor']);
+    const { backgroundColor: _, ...newElementWithoutBgColor } = newElement;
```

Comment:
Why rename it to `_` if unused anyway?

Reply:
ESLint complained otherwise, surprisingly